app/code/Magento/InventoryIndexer/Plugin/InventoryApi/ReindexAfterSourceItemsSavePlugin.php

```diff
@@ -51,13 +49,7 @@ public function afterExecute(
         $result,
         array $sourceItems
     ) {
-
-        $sourceItemIds = [];
-        foreach ($sourceItems as $sourceItem) {
-            // TODO: replace on multi operation
-            $sourceItemIds[] = $this->getSourceItemId->execute($sourceItem->getSku(), $sourceItem->getSourceCode());
-        }
-
+        $sourceItemIds = $this->getSourceItemIds->execute($sourceItems);
```

Comment:
Do we really need this workaround? Why we can't just add getId() method to Magento\InventoryApi\Api\Data\SourceItemInterface?

Reply:
@nmalevanec unused class removed